File: training-data/java/IsPrime.java
import java.util.ArrayList;
import java.util.List;

public class IsPrime {
    public static boolean isPrime(int n) {
        if (n < 2) return false;
        if (n == 2 || n == 3) return true;
        if (n % 2 == 0) return false;
        for (int i = 3; (long) i * i <= n; i += 2) {
            if (n % i == 0) return false;
        }
        return true;
    }

    public static void main(String[] args) {
        List<Integer> primes = new ArrayList<>();
        for (int n = 0; n < 20; n++) {
            if (isPrime(n)) primes.add(n);
        }
        System.out.println(primes);
    }
}

File: training-data/javascript/is_prime.js
function isPrime(n) {
  if (n < 2) return false;
  if (n === 2 || n === 3) return true;
  if (n % 2 === 0) return false;
  for (let i = 3; i * i <= n; i += 2) {
    if (n % i === 0) return false;
  }
  return true;
}

const primes = Array.from({ length: 20 }, (_, i) => i).filter(isPrime);
console.log(primes);

module.exports = { isPrime };
